src/providers/aws.rs

```diff
@@ -957,10 +988,19 @@ impl RegionLauncher {
         self.instances
             .values()
             .map(|info| match info {
-                (Some((public_ip, public_dns)), (name, Setup { username, .. })) => {
+                TaggedSetup { 
+                    name,
+                    setup: Setup { username, .. },
+                    ipinfo: Some(IpInfo {
+                        dns: public_dns,
+                        pub_ip: public_ip,
+                        priv_ip: private_ip,
```

Comment:
Why not just name these with the long names so you wouldn't have to do things like this? :p

Reply:
fair enough :)